app/renderer/js/electron-bridge.ts

```diff
@@ -46,6 +47,28 @@ class ElectronBridge extends EventEmitter {
 	set_send_notification_reply_message_supported = (value: boolean): void => {
 		this.send_notification_reply_message_supported = value;
 	};
+
+	decrypt_clipboard = (): { key: string; check(): string | null } => {
+		const key = crypto.randomBytes(32);
```

Comment:
I think probably it's worth a block comment here explaining the interface of this, namely that it provides a randomly generated key as well as a function to decrypt content encrypted using that key; and the encryption algorithm used by the server needs to be the same as what is used here.  

I sorta wonder if we should have this present a third parameter, `algorithm_version`, that is an int that gets passed on to the server; this would be setup for if we decide to change the algorithm in the future, the server could know how the app is expecting us to encrypt.  I suppose we could always add this later with "didn't send" meaning v0, but it'd be pretty cheap to just have an `algorithm_version: 1` here so that the server-side code can assume that field is always present.

Reply:
Done.